### ISSUE TITLE
Requests/Offers Pages Headers + Divisions

### ISSUE THREAD

**Issue by joatu** [joatuapp/joatu-v2#84]
Considering that there are three columns in the general offers/requests sections, I they should be divided by the three headlines you came up with.

* Skills And Time
* Physical Goods
* Knowledge

**Comment by undergroundwebdesigns** [comment]
I disagree. Firstly, the 3 columns is arbitrary, and I've been seriously considering changing it. Also, what if we want to change the number of offer/request types? If we add more (5, 6) columns won't work at all. It's also a very poor use of the page, if there's say 10 requests of one type, and none of the others, we'd have a large blank space.

That said, I agree the offer / request listing page could use some work. And I'm not against some indication of the type of offer / request... we could include it in the individual offer section, or maybe colour-code the offers? Lots of ideas....

**Comment by joatu** [comment]
Color coding is good too.  Ordering is good. I liked the idea of the columns because I didn't see us extending the different types of offers, just expanding upon each of them.  For example, a physical good can be rented/loaned/sold/given away.  So when you click physical items, you can click on each one of those to see which fall into each category.

Right now it's obviously not too appealing to go to the offers page and read through the offers. I like finding ways to order them that would make it appealing to look through.

And at some point, a tinder-like version where people can flip through offers that are locally available according to their location.  Complete aside, but just wanted to share that thought on different ways people will skim items.

I support color-coding & some sort of ordering (which is why I initially suggested columns).  How the ordering should work...I'm totally up for discussion.

**Comment by undergroundwebdesigns** [comment]
It sounds like you want search filters, and ordering. Ordering is usually more about quantifiable things though, such as time (order by newest) or location. I am definitely down for adding some filters to the offer / request listing pages so that people can, for example, show _just_ physical goods, or _just_ local (vs global) offers, or other combinations of things. What types of filters (besides those two) would you want?

**Comment by joatu** [comment]
Yeah, the pop-up is the dream. :cactus: So many strange symbols, I never knew.

Local/Global, Just Physical Goods, Just Education.

Later: Just CAPS as an option.  For Rent/Sale/Free/etc.

**Comment by joatu** [comment]
I think we can put this on the backburner, because it has some good thoughts/comments (e.g. pop-up) but it isn't crucial at this point in time.

**Comment by joatu** [comment]
We need to discuss the parameters for offers. For example, having Goods/Skills/QuickKnowledge where the third option could allow a user to check-off 'phone' 'google hangouts' 'skype' and possibly even typical availabilities that someone requesting could send them a proposition from the very first message and with a click, the other person could accept it and have their skype or phone number shared.

In the meantime, I think removing 'Knowledge' might be for the best. For simplicity's sake.
